CMakeLists.txt

```diff
@@ -163,5 +166,22 @@ if(SOCI_TESTS)
   add_subdirectory(tests)
 endif()
 
+###############################################################################
+# build config file
+###############################################################################
```

Comment:
On 5 Nov 2015 15:06, "ArnaudD-FR" <notifications@github.com> wrote:
>
> In CMakeLists.txt:
>
> > @@ -163,5 +166,22 @@ if(SOCI_TESTS)
> >    add_subdirectory(tests)
> >  endif()
> >
> >
+###############################################################################
> > +# build config file
> >
+###############################################################################
>
> Now I handle only cache variable starting with SOCI_HAVE_. I'm using
CACHE_HELPSTRING property to add some help in generated header
>

Looks good to me. I'll merge when I'm back home or @snikulov feel free to
do it if you see no objections.

Thank you!